pkg/sql/colexec/semi/join.go

```diff
@@ -276,7 +276,7 @@ func (ctr *container) probe(bat *batch.Batch, ap *SemiJoin, proc *process.Proces
 	}
 
 	for j, pos := range ap.Result {
-		if err := ctr.rbat.Vecs[j].PreExtendArea(len(bat.Vecs[pos].GetArea()), proc.Mp()); err != nil {
+		if err := ctr.rbat.Vecs[j].PreExtendArea(len(eligible), len(bat.Vecs[pos].GetArea()), proc.Mp()); err != nil {
```

Comment:
PreExtendArea -> PreExtendWithArea

Reply:
How is that possible ...    The code compiled!   :(

Sorry, fixing now.